ddtrace/appsec/_iast/_evidence_redaction/_sensitive_handler.py

```diff
@@ -16,7 +16,7 @@
 
 log = get_logger(__name__)
 
-REDACTED_SOURCE_BUFFER = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+REDACTED_SOURCE_BUFFER = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789" * 20
```

Comment:
I would prefer a function that receives the length we want and returns the string with that length, what do you think?

Reply:
Something like:

```
def get_redacted_source(length):
    repeat_string = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
    full_repeats = length // len(repeat_string)
    remainder = length % len(repeat_string)
    result = repeat_string * full_repeats + repeat_string[:remainder]
    return result
```